paddle/fluid/framework/details/build_strategy.h

```diff
@@ -74,8 +74,14 @@ struct BuildStrategy {
 
   bool fuse_elewise_add_act_ops_{false};
 
+  bool fuse_all_reduce_ops_{false};
```

Comment:
these 3 flags are not needed.

Reply:
We need those flags currently. When we test those pass on more models and ensure that those pass can make program faster, we can set the default value as True.